File: app/src/main/java/com/cookandroid/torchapp/SettingsFragment.java
package com.cookandroid.torchapp;

import android.os.Bundle;
import android.view.ViewGroup;

import androidx.preference.PreferenceFragmentCompat;
import androidx.preference.SeekBarPreference;

public class SettingsFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.root_preferences, rootKey);
    }
}

File: app/src/main/java/com/cookandroid/torchappkt/SettingsFragment.kt
package com.cookandroid.torchappkt

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.cookandroid.torchappkt.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.root_preferences, rootKey)
    }
}